src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -1227,14 +1238,6 @@ private void CheckForShutdown()
 
             _connectionWindow.Dispose();
             _concurrentStreams.Dispose();
-
-            // ISSUE #35466
```

Comment:
Why was this comment removed? We haven't actually fixed the issue here, have we?

Reply:
I think the flow is understood and remaining work tracked by other existing issues. If you have specific 3.0 task let me know or we can keep it open for future investigation.